### DISCUSSION TITLE
Charges for an NGO using organisation with mutiple projects and teams

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3907] [Q&A]
I would like to know the pricing that would be charged to an NGO "Asha for education, Mumbai, India" 
A brief introduction to what we are doing : We are actively approaching colleges and schools for "Learning Digital Geography for Sustainability", where Qfield and Qfield cloud is one of the tools that is taught. Students are registered as individual collaborators or are part of a team. Project on cloud is created for the admin boundary surrounding their area of school /college. 
So my queries are based on these parameters:
1. there are more than 6-7 projects inside this organisation . New project evolves as new school/college join us.

2. It is not sure how many students will take active  part in the project. Number of students actively using may also vary within a month and it will be like 2 students are working for a week then they are having exams , so not working ...then they may start working again.

So how will the NGO be charged ?
 Will there be additional discount for an NGO apart from 40% what you are offering currently?
 Will it be only on monthly basis or is there any provision for yearly discounts as well?

**Comment by lucienicolier** [comment]
Hi,

Thanks for your message.

Can you email info@opengis.ch and ask for an offer for an NGO.

Best

**Comment by MonikaShah** [comment]
Sure
